src/libstd/sys/common/remutex.rs

```diff
@@ -205,6 +204,7 @@ mod tests {
     }
 
     #[test]
+    #[allow(unused_variables)]
```

Comment:
I think removing the unused variables here would change the test, because the locks are stored in variables to extend their lifetime to the end of the block, right?